src/pages/hyoeunJung/Main/Main.scss

```
@@ -0,0 +1,150 @@
+* {
+  box-sizing: border-box;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  height: 50px;
+  align-items: center;
+  border-bottom: 1px solid rgb(205, 205, 205);
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: white;
+  margin: 0 auto;
+  .navInput {
+    height: 20px;
+  }
+}
```

```suggestion
.nav {
  width: 100%;
  display: flex;
  justify-content: space-around;
  height: 50px;
  align-items: center;
  border-bottom: 1px solid rgb(205, 205, 205);
  position: fixed;
  top: 0;
  left: 0;
  background-color: white;
  margin: 0 auto;
  
  .navInput {
    height: 20px;
  }
}
```
className 선택자 바로 위 한 줄은 개행시켜서 가독성을 증가시켜주세요!
모든 css 파일의 모든부분에 꼭 적용시켜주세요!